homeassistant/components/sensibo/strings.json

```
@@ -351,6 +353,7 @@
               "quiet": "Quiet",
               "strong": "Strong",
               "low": "[%key:component::climate::entity_component::_::state_attributes::fan_mode::state::low%]",
+              "medium_low": "Medium low",
```

```suggestion
              "medium_low": "[%key:component::sensibo::entity::sensor::climate_react_low::state_attributes::fanlevel::state::medium_low%]",
```
Any reason we are excluding this?

---

No, but I saw there are more or those so I would prefer to get this now and do it in follow-up PR to "clean" strings in one go.